CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+
+* For documentation issues, please first post an issue describing what you propose to add, where you’d like to add it, and a description of why it is an important addition/ For documentation improvements and fixes, you should post an issue of what is currently wrong or missing, and how you propose to address it. 
+
+When you post your issue, the American Gut developers will respond to let you know if we agree with the addition or change. It’s important to go through this step to avoid wasting time working on a feature that we are not interested in including in scikit-bio. 
+
+## Code Review
+WHen you submit code in American Gut, it will be reviewed by one or more American Gut developer. These reviews are intended to confirm a few points:
+* Your code is sufficiently well tested (see Testing Guidelines below)
+* Your analyses adheres to submission guidelines (see IPython Notebooks below)
+* Your code and analysis are sufficiently well documented
+* Your update provides relevant changes or additions.
+
+This process is designed to ensure the quality of American Gut submissions, and can provide useful experience for new developers.
+
+For big changes, if you’d like feedback on your code as you work, you should request help in the issue that you created, and one of the American Gut developers will work with you to perform regular code review. 
+
+## Submitting Code to American Gut
+American Gut is hosted on [GitHub](http://www.github.com), and we use GitHub's [Pull Request](https://help.github.com/articles/using-pull-requests) mechanism for accepting submissions. You should go through the following steps to submit code or analyses to American Gut.
+
+* Begin by [creating an issue](https://github.com/biocore/american-gut/issues) describing your proposed analysis or change. This should include a description of the proposed change, and a note in the issue documentation that you'd like to work on it. Once you hear back from a maintainer that it's okay to make the change, we will assign the issue to you.
+2. [Fork](https://help.github.com/articles/fork-a-repo) the American Gut repository on the Github Website to your Github account.
+3. Clone your forked repository to the system where you'll be developing using `Git clone`
+4. Ensure that you have the latest version of all the files (especially important if you cloned a long time ago). You should do this by adding American Gut as a remote repository and then pulling from that repository. You'll only need to run the `git remote` step one time:
+```
+git checkout master
+git remote add upstream master http://github.com/biocore/american-gut.git
+git pull upstream master
+```
+5. Create a new topic branch that you will use to make your changes with `git checkout -b`:
```

Comment:
this renders as `1`, the mixed use of numbers and bullets in this section seems to be messing the renderer